lib/y2logsstep.pm

```diff
@@ -279,8 +279,9 @@ sub save_strace_gdb_output {
     return if (get_var('NOLOGS'));
 
     # Collect yast2 installer or yast2 module trace if is still running
-    if (!script_run(qq{ps -eo pid,comm | grep -i [y]2start | cut -f 2 -d " " > /dev/$serialdev}, 0)) {
+    if (!script_run(qq{ps -eo pid,comm | grep -i "[y]2start installation " | cut -f 2 -d " " > /dev/$serialdev}, 0)) {
```

Comment:
Actually I would not introduce this change, as it worked perfectly for yast modules straight away. And as it's not directly related to the initial PR, maybe let's put it in separate one. Other changes LGTM.

Reply:
So do you want to keep it for all yast2 modules ? I thought that we are aiming for the installer only.

Reply:
As a part of some ticket, I've reused it in `lib/console_yasttest.pm`. So, in case you want to introduce this change, you could use `$is_yast_module` flag to differentiate between installer and yast module. But this code worked smoothly.

Reply:
oh, sorry!!! Honestly I have not read the whole function, thus I have missed this flag. My bad! Ok, so I will leave only the check for non empty wait_serial output -> ``return unless defined($yast_pid);`` It does not make sense to continue in the code as long as we have no PID. Thanks!

Just a note regarding flag ``$is_yast_module``, do you plan to use it as boolean variable or with more states ? I have noticed that you are passing a string  as argument of the function in ``console_yasttest.pm``.

Reply:
as of now it's just flag, ideally we could pass module name to be more precise when looking for pid, but I left it for later.

Reply:
ok, so I have added ``my $opt = defined($is_yast_module) ? 'module' : 'installer';`` to avoid the confusion of named assets.

Reply:
Great, thanks! Let's keep it open a bit to give a chance to others to review. If no additional points are raised - I will merge

Reply:
Reasonable, thanks!